src/app/games/mancala/kalah/KalahTutorial.ts

```diff
@@ -0,0 +1,83 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { MancalaState } from '../commons/MancalaState';
+import { KalahMove } from './KalahMove';
+import { MancalaDistribution } from '../commons/MancalaMove';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+import { MancalaTutorial } from '../commons/MancalaTutorial';
+
+export class KalahTutorial extends Tutorial {
+
+    public gameName: string = $localize`Kalah`;
+
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            this.gameName,
+            $localize`${this.gameName} is a Mancala. Mancala is the name of a family of board games that date back at least to the third century. Mancalas are games of distribution (sowing) and capture. Their goal is to capture the most seeds. The spaces in Mancalas are called the houses. As you're playing Dark, the 6 houses on the bottom are yours.`,
+            MancalaState.getInitialState(),
+        ),
+        TutorialStep.informational(
+            $localize`Kalah`,
+            $localize`Bonus fact: Kalah has been created in the U.S.A in 1940 by William Julius Champion Jr.`,
+            MancalaState.getInitialState(),
+        ),
+        MancalaTutorial.SOWING(KalahMove.of(MancalaDistribution.FIVE)),
+
+        TutorialStep.forClick(
+            $localize`The Kalah (1/2)`,
+            $localize`The houses on the extreme left and right, unaligned to the others, are the Kalah. Yours is on the left, the opponent's on the right. When sowing, before passing from your last house to the first of the opponent, you must drop one seed in your Kalah. But you won't have to drop seed in your opponent's Kalah. When you make a capture, the captured seeds are put in your Kalah.<br/><br/>You're playing Dark. Make a move that pass through your Kalah then feed opponent's houses.`,
```

Comment:
"But" n'est pas un début de phrase valide. Tu peux juste le drop, ou joindre les deux phrases

Reply:
make a move that *passes*

Reply:
then feeds ok